recipes/hdf5/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: hdf5
+  version: 1.8.16
+
+source:
+  fn: hdf5-1.8.16.tar.gz
+  url: http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-1.8.16.tar.gz
+  md5: b8ed9a36ae142317f88b0c7ef4b9c618
+
+
+build:
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - python    # [win]
```

Comment:
My rule of thumb is:

  - if it's a shared library that something other than Python will use, you only need python in build to activate features.
  - if it is a shared library that is tied to a Python API (e.g. Boost Python), you need BOTH.
  - If it is a library that has compiled content, but is used only from Python, you need Python in both build and runtime requirements, but you do NOT need the features block.